findimports.py

```diff
@@ -1001,10 +1053,18 @@ def main(argv=None):
                          help="ignore a file or directory;"
                               " this option can be used multiple times."
                               " Default: ['venv']")
+    options.add_argument('-R', '--rmprefix', metavar="PREFIX", nargs="+",
+                         help="remove PREFIX from displayed node names. "
+                              "This operation is applied last. "
+                              "Names that collapses to nothing are removed.")
+
     try:
         args = parser.parse_args(args=argv[1:] if argv else None)
     except SystemExit as e:
         return e.code
+    if args.condense_to_packages and args.condense_to_packages_externals:
+        raise argparse.ArgumentError(
+            None, 'only one of -p and -pE can be provided')
```

Comment:
This is the only statement not covered by the unit tests.

Reply:
Should be good now.